### PR TITLE
Class-ify 'aws_acm_list'

### PR DESCRIPTION
**Pull request by dfellis**
Part of #862

### CODE REVIEW

File: src/modules/0.0.17/aws_acm_import/index.ts [hunk 1]
```diff
@@ -29,6 +29,19 @@ class CertificateImportMapper extends MapperBase<CertificateImport> {
...
+    delete: (es: CertificateImport[], ctx: Context) => ctx.orm.remove(CertificateImport, es),
+    read: async (ctx: Context, id?: string) => {
+      const opts = id ? {
+        where: {
+          id,
+        }
+      } : {};
+      return await ctx.orm.find(CertificateImport, opts);
+    },
+  });
```
Comment (dfellis) [inline src/modules/0.0.17/aws_acm_import/index.ts:44]: Due to a bug in how we were detecting `@cloudId` vs `entityId` usage, these were being autogenerated *incorrectly* before, and only worked for our usage by chance.

Fixed that bug and now added these in.

File: src/modules/interfaces.ts [hunk 2]
```diff
@@ -299,7 +299,7 @@ export class MapperBase<E> {
       this.db.entityId = this.entityId;
       this.db.dest = 'db';
       this.db.module = this.module;
-    } else if (!!cloudColumn) {
+    } else if (!!cloudColumn && !((cloudColumn as any) instanceof Error)) {
```
Comment (dfellis) [inline src/modules/interfaces.ts:302]: This is the bugfix. `getCloudId` doesn't return `undefined` when not specified, it returns an `Error` object instead, which is truthy.

**Review by depombo — APPROVED** [review]

File: src/modules/interfaces.ts (hunk 2 above)
Comment (aguillenv) [reply src/modules/interfaces.ts:302]: how did you catch it??

Comment (dfellis) [reply src/modules/interfaces.ts:302]: When converting this module I forgot to include the `module` property on the mapper and was getting errors that didn't make sense, so I read through the initialization logic *and* the functions it calls, noticing this error and fixing it but it didn't fix the bug I was seeing, and then I finally realized what I had done wrong during the conversion process.

**Pull request merged**